pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -143,6 +143,11 @@ func (c *Cluster) makeDeployment(nodeName string, devices []rookalpha.Device, se
 
 	commonArgs := []string{
 		"--foreground",
+		"--log-file",
```

Comment:
Instead of adding this option, how about if we set it to empty in the config file so it can still be overridden? As is, users would be prevented from overriding the setting to collect the log in a file, right?

Reply:
That's correct. That just requires auditing/updating all the users of the option.